app/controllers/stats_controller.rb

```diff
@@ -74,6 +83,46 @@ def generate_commit_csv
     end
   end
 
+  def generate_translation_words_csv
```

Comment:
all of this logic should live outside of the controller. Probably in a file in `/lib/reports/`